### DISCUSSION TITLE
Nuclei skips the active ip from scanning problem

### DISCUSSION THREAD

**Discussion by Rameshkanna2k** [projectdiscovery/nuclei#5177] [Q&A]
Hi team,

While scanning the target with nuclei using

nuclei -u 129.33.40.139:443 -s low,medium,high,critical

its not working and target skipped 
and also 

nuclei -u https://129.33.40.139 -s low,medium,high,critical

not working and target skipped

While reaching the ip with broswer using

https://129.33.40.139 ---> working
129.33.40.139:443  ------->not working

i have attached the screenshots.
Kindly help me to fix my issue , why nuclei skippping the target . is their anything i need to fix from my network settings or in broswer or nuclei.
![Screenshot from 2024-05-13 11-11-40](https://github.com/projectdiscovery/nuclei/assets/67623153/e2335fbf-3af7-4f1b-acf2-387c5adca2e7)
![Screenshot from 2024-05-13 11-12-22](https://github.com/projectdiscovery/nuclei/assets/67623153/cccf27b7-b955-4c9f-89b0-1f7f2f86b3f8)
![Screenshot from 2024-05-13 11-10-31](https://github.com/projectdiscovery/nuclei/assets/67623153/247b7345-bb33-4d93-aff2-3059082a0756)

**Comment by ehsandeep** [comment]
this is now being tracked as issue at https://github.com/projectdiscovery/nuclei/issues/5179
